examples/bulb/js/dan2.js

```diff
@@ -202,6 +204,8 @@ var dan2 = (function () {
     return {
         'register': register,
         'push': push,
-        'UUID': UUID,
+        'UUID': function() {
+            return _id;
+        },
     };
-})();
+})();
```

Comment:
My sublime solve it automatically while working with [620a9d: Try to fix parameters inconsistency](https://github.com/IoTtalk/iottalk-js/pull/2/commits/620a9d5bbf54b8eec3f85bc2fd545dea32280d04)